src/app/vault/ciphers.component.html

```diff
@@ -41,7 +41,7 @@
   <div class="no-items" *ngIf="!ciphers.length">
     <i class="fa fa-spinner fa-spin fa-3x" *ngIf="!loaded" aria-hidden="true"></i>
     <ng-container *ngIf="loaded">
-      <i class="fa fa-frown-o fa-4x" aria-hidden="true"></i>
+      <img class="no-items-image" />
```

Comment:
Same, `aria-hidden="true"` is probably best here, but at the least an alt text would be good.

Reply:
Fixed!